libraries/api.rb

```diff
@@ -69,7 +63,7 @@ def request(method, path, content_type = 'application/json', data = nil)
 
     # Runs a specific script with parameters
     def run_script(scriptname, params)
-      body = request(:post, "script/#{scriptname}/run", 'text/plain', params)
+      body = request(:post, "script/#{scriptname}/run", data: params)
```

Comment:
Why do you remove the `text/plain` content type? shouldn't you hae `content_type: 'text/plain'`

Reply:
Our scripts except JSON, and with the restriction on content-type to generate json or not, if we keep text/plain, our scripts cannot parse it. And Nexus does not seem to care, so better be explicit / clean on what we send.